test/odbc/psqlodbc/bytea.cpp

```diff
@@ -0,0 +1,377 @@
+#include "psqlodbc_tests_common.h"
+
+const string BBF_TABLE_NAME = "master.dbo.bytea_table_odbc_test";
+// For BBF Connection
+//   Cannot prepend database name when creating/dropping view
+//   Must prepend database name when selecting from view
+const string BBF_VIEW_NAME = "dbo.bytea_view_odbc_test";
+const string PG_TABLE_NAME = "master_dbo.bytea_table_odbc_test";
+const string PG_VIEW_NAME = "master_dbo.bytea_view_odbc_test";
+const string COL1_NAME = "pk";
+const string COL2_NAME = "data";
+const string DATATYPE_NAME = "bytea";
+const vector<pair<string, string>> TABLE_COLUMNS = {
+  {COL1_NAME, "INT PRIMARY KEY"},
+  {COL2_NAME, DATATYPE_NAME}
+};
+
+class PSQL_DataTypes_ByteA : public testing::Test {
+  void SetUp() override {
+    if (!Drivers::DriverExists(ServerType::PSQL)) {
+      GTEST_SKIP() << "PSQL Driver not present: skipping all tests for this fixture.";
+    }
+    if (!Drivers::DriverExists(ServerType::MSSQL)) {
+      GTEST_SKIP() << "MSSQL Driver not present: skipping all tests for this fixture.";
+    }
+
+    OdbcHandler bbf_test_setup(Drivers::GetDriver(ServerType::MSSQL));
+    bbf_test_setup.ConnectAndExecQuery(DropObjectStatement("TABLE", BBF_TABLE_NAME));
+
+    OdbcHandler pg_test_setup(Drivers::GetDriver(ServerType::PSQL));
+    pg_test_setup.ConnectAndExecQuery(DropObjectStatement("TABLE", PG_TABLE_NAME));
+  }
+
+  void TearDown() override {
+    if (!Drivers::DriverExists(ServerType::PSQL)) {
```

Comment:
Is this needed in the TearDown? Seems redundant

Reply:
This is required as `TearDown()` will run even if `SetUp()` says to skip the test.